### DISCUSSION TITLE
where did you find icons such as arch and flatpak?

### DISCUSSION THREAD

**Discussion by l4v3nx** [end-4/dots-hyprland#531] [Q&A] [answered]
i'm going to write more quick scripts in sideleft and i wonder where you found these icons. i checked https://fonts.google.com/icons and i didn't find such icons as arch-symbolic and flatpak-symbolic. i tried to insert some icons from this website (vpn-key and vpn-key-off) and for some reason it showed the blank icon.

p.s. sorry for bad english))))

**Comment by end-4** [comment]
**_There are two different types of icons: [Material Symbols] and [normal icons (commonly for apps)]_**
For "normal icons", they're in your icon pack and in the `.config/ags/assets/icons` folder. svg files there are available for use with `Widget.Icon()`
For Material Symbols, you apply them by putting text and set the font to that icon font. For my dotfiles, just add the `icon-material` class to do this.

**Reply by l4v3nx** [reply]
you see, here https://github.com/end-4/dots-hyprland/blob/66b30318cd6cda6a8c48b450416af0ebd2f7b289/.config/ags/modules/sideleft/tools/quickscripts.js#L70-L72 is the scss class called `sidebar-module-btn-icon`, but i didn't find this class among all of your dots. I renamed it to `sidebar-module-btn-arrow` (i guess it's not much visually different). I tried changing class to icon-material, but it still shows svg icons and blank material icon. (there is also the MaterialIcon function, but i didn't find where and how to put it). Can you tell me what i am doing wrong?

**Reply by l4v3nx** [reply]
as for the script itself, it should be like this
```
    {
        icon: 'vpn_key',    // https://fonts.google.com/icons?selected=Material+Symbols+Outlined:vpn_key:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=vpn+key&icon.size=24&icon.color=%23e8eaed
        name: 'Turn on AmneziaWG tunnel',
        command: `sudo awg-quick up awg-nl`,
        enabled: isArchDistro,
    },
```
(btw it would be nice to set `distroID` to any distro and not only arch)

**Reply by end-4** [reply, marked as answer]
oh, the icon for those scripts use svgs, so you can't simply have a material icon name there

> (btw it would be nice to set distroID to any distro and not only arch)

just, `enabled: true`

**Reply by l4v3nx** [reply]
so the easiest way to add these icons is to download svgs from https://fonts.google.com/icons ?

also what do you think about `sidebar-module-btn-icon` and `sidebar-module-btn-arrow`?
